rpcs3/Emu/RSX/gcm_printing.cpp

```diff
@@ -1111,28 +1089,47 @@ namespace
 
 	namespace
 	{
+
+		template <u32 opcode, typename = void>
```

Comment:
Please use Camel case for named template parameters.

Reply:
Fixed.